docs/content/docs/bundlers.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Fluid and Bundlers
+aliases:
+  - "/start/bundlers/"
+---
+
+The following are known bundler issues, answers to some of the most commonly asked questions about the Fluid
+Framework and Bundlers.
+
+## Known Errors with Webpack 5
+We know the following packages cause errors; assert, buffer, events and url.
```

Comment:
It would probably be worth adding a brief introductory statement noting that these may be encountered when migrating from Webpack 4 to Webpack 5. The target audience of this doc is specifically users who are already using us with Webpack v4 and are upgrading.

Reply:
> The target audience of this doc is specifically users who are already using us with Webpack v4 and are upgrading.

Not necessarily. The people I am most concerned about are those that are adding Fluid to an existing project or using a scaffolding tool like create-react-app. People that are upgrading Webpack 4 -> 5 know about polyfills, usually. People starting fresh with Webpack 5 often expect it to "just work."

Reply:
Is there anything else to do here or should I mark this as resolved?

Reply:
I think we're good then. I misunderstood the situation 🙂